docker/images/proxysql/rhel-compliant/entrypoint/entrypoint.bash

```
@@ -81,13 +81,50 @@ cp -a etc/proxysql.cnf proxysql/etc/
+# Belt-and-braces: the spec gates `/usr/lib/proxysql/*.so` under
+# `%if 0%{?with_plugins}`, but rpmbuild aborts with "File not found
+# by glob" if the directory exists with no .so files.  This can
+# happen if a plugin build silently produced no artefact (link
+# failure that returned 0, etc.).  Only set with_plugins=1 when at
+# least one .so actually made it to the staging directory.
+RPMBUILD_WITH_PLUGINS=0
+if compgen -G "proxysql/usr/lib/proxysql/*.so" >/dev/null 2>&1; then
+    RPMBUILD_WITH_PLUGINS=1
+fi
 mv proxysql "proxysql-${CURVER}"
 tar czvf "proxysql-${CURVER}.tar.gz" proxysql-${CURVER}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)